ftml/docs/README.md

```diff
@@ -0,0 +1 @@
+../README.md
```

Comment:
Curious - what's this link here for?

Reply:
I guess since it's documentation. It's not really necessary, I just thought I'd add it.